frontend/frontend-nav-menu.php

```diff
@@ -115,7 +115,7 @@ public function wp_get_nav_menu_items( $items ) {
 				/** This filter is documented in include/switcher.php */
 				$options = apply_filters( 'pll_the_languages_args', $options ); // Honor the filter here for 'show_flags', 'show_names' and 'dropdown'.
 
-				$switcher = PLL_Switcher::create( $this->polylang );
+				$switcher = new PLL_Frontend_Switcher();
```

Comment:
It seems we do not need the `$polylang` property anymore then:

https://github.com/polylang/polylang/blob/2e0eb9c1ea34e74678329a722ea1840c617821e0/frontend/frontend-nav-menu.php#L24-L36

Reply:
Done in the latest commit 👍